backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisWebservice.java

```diff
@@ -55,6 +56,8 @@ public static void start(int port) {
     get(("/" + OIDC_LOGIN_PATH), oidcController::handleLoginRequest);
     get("/" + ROBOTS_TXT, MolgenisWebservice::robotsDotTxt);
 
+    get("/:schema/sitemap.xml", SiteMapController::getSiteMapForSchema);
```

Comment:
Can you please use same patter as all other *Api classes, and use a 'create' function to create these things?

Reply:
why the extra indirection , this is a class that has only a single function / method , it is a static class with a static method. Maybe discuss this offline , i would prefer non static singleton classes that have there deps injected for easy testing

Reply:
two considerations: (1) consistency with what is already there, so if we change this we do it everywhere and (2) reason why we use 'create()' is such that all code related to this api is in same place. Because there is a tight coupling between these two pieces of code because of the variable mapping.